raiden/network/protocol.py

```diff
@@ -307,6 +308,15 @@ def healthcheck(
 
     """ Sends a periodical Ping to `receiver_address` to check its health. """
 
+    if log.isEnabledFor(logging.DEBUG):
+        log.debug(
+            'starting healthcheck for',
+            node=pex(protocol.raiden.address),
+            to=pex(receiver_address),
+            current_state=NODE_NETWORK_UNKNOWN,
```

Comment:
since `current_state` and `new_state` are  constant here, always unknown why include them in the debug log? I think we should keep logs as succinct as possible. We will thank ourselves later during debugging.

Reply:
the state changes in the loop bellow, added in the first two logging calls to make them consistent

Reply:
yes I saw, but it does not change in this point in code or in the other one I pointed out, right? Here consistency does not matter as much as having smaller log lines does.

Reply:
why consistency doesnt matter?

Reply:
because consistency in the code is no as important as having smaller, more readable log lines

Reply:
I disagree, consistent log lines are important for the structure logging / filtering. But, that subject is so minor I'm just going to change it.